# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../flac/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- flac-1.2.1/examples/cpp/encode/file/main.cpp.vanilla	2008-04-09 10:32:49.000000000 +0000
+++ flac-1.2.1/examples/cpp/encode/file/main.cpp	2008-04-09 10:32:55.000000000 +0000
@@ -30,6 +30,7 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 #include "FLAC++/metadata.h"
 #include "FLAC++/encoder.h"
 
